app/javascript/dashboard/routes/dashboard/settings/labels/AddLabel.vue

```diff
@@ -41,6 +43,7 @@
           <woot-button
             :is-disabled="$v.title.$invalid || uiFlags.isCreating"
             :is-loading="uiFlags.isCreating"
+            data-testId="label-submit"
```

Comment:
Can we follow consistent naming `testId` or `testid` everywhere?

Reply:
@nithindavid fixed the typo and updated it to `data-testid`